### PR TITLE
Some fixes + update ROR + more readme

### PR DESCRIPTION
**Pull request by sergeyf**

### CODE REVIEW

File: scripts/update_openalex_works_counts.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+s3 = boto3.client("s3")
+paginator = s3.get_paginator("list_objects_v2")
+pages = paginator.paginate(Bucket=bucket, Prefix=prefix)
+
+works_count_dict = {}
+for page in pages:
+    for obj in page["Contents"]:
+        if obj["Key"].startswith("part") and obj["Key"].endswith(".gz"):
+            print("Workong on", obj["Key"])
```
Comment (Whattabatt) [inline scripts/update_openalex_works_counts.py:31]: typo

File: s2aff/consts.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
 PATHS = {
     "ner_training_data": os.path.join(PROJECT_ROOT_PATH, "data", "ner_training_data.pickle"),
-    "ror_data": os.path.join(PROJECT_ROOT_PATH, "data", "v1.1-2022-06-16-ror-data.json"),
+    "ror_data": os.path.join(PROJECT_ROOT_PATH, "data", "v1.18-2023-01-26-ror-data.json"),
```
Comment (Whattabatt) [inline s2aff/consts.py:18]: would be a bit cleaner to have a ROR_VERSION variable

File: setup.py [hunk 3]
```diff
@@ -7,9 +7,9 @@
 
 setuptools.setup(
     name="s2aff",
-    version="0.11",
+    version="0.12",
```
Comment (Whattabatt) [inline setup.py:10]: This is already 0.12 in main https://github.com/allenai/S2AFF/blob/main/setup.py, so should prob be 13

**Review by Whattabatt — COMMENTED** [review]
Only version bump is blocking

File: scripts/update_openalex_works_counts.py (hunk 1 above)
Comment (sergeyf) [reply scripts/update_openalex_works_counts.py]: Fixed

File: s2aff/consts.py (hunk 2 above)
Comment (sergeyf) [reply s2aff/consts.py]: Good idea, fixed

File: setup.py (hunk 3 above)
Comment (sergeyf) [reply setup.py]: Ooops, fixed.

**Pull request merged**